Tutorials/python/samtools_count_para_chr_subprocess_py/Readme.md

```diff
@@ -8,7 +8,7 @@ In order to take full advantage of the scalability that cloud computing offers,
 This applet tutorial code is similar to the [_Parallel Cores SAMtools count tutorial_](/python_parallel_tutorial.html#samtools_count_para_reg_subprocess_py), except `multiprocessing.dummy` is used instead of `multiprocessing`.
```

Comment:
In this file the link "multiprocessing.dummy" should point to:

https://docs.python.org/3/library/multiprocessing.html#module-multiprocessing.dummy

Reply:
done